bindings/rust-bindings/etc/configs.toml

```diff
@@ -676,7 +676,7 @@
       Challenger = "0x45eFFbD799Ab49122eeEAB75B78D9C56A187F9A7"
       Proposer = "0x0000000000000000000000000000000000000000"
       UnsafeBlockSigner = "0x0000000000000000000000000000000000000000"
-      BatchSubmitter = "0x0000000000000000000000000000000000000000"
+      BatchSubmitter = "0xdb80Eca386AC72a55510e33CF9CF7533e75916eE"
```

Comment:
This address was omitted from the original config. Added it after the retrieving the value onchain. We need this value to be set in order to perform the da availability test added by this pr